lib/internal/child_process.js

```diff
@@ -230,12 +230,12 @@ util.inherits(ChildProcess, EventEmitter);
 
 
 function flushStdio(subprocess) {
-  if (subprocess.stdio == null) return;
-  subprocess.stdio.forEach(function(stream, fd, stdio) {
-    if (!stream || !stream.readable || stream._readableState.readableListening)
-      return;
-    stream.resume();
-  });
+  if (subprocess.stdio === null || subprocess.stdio === undefined) return;
```

Comment:
I'm not sure this change is really worth it, the previous conditional is shorter and is equivalent in behavior.